### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Link New HDInsight Cluster_Keyboard: Keyboard trap is observed in the multi line text field "Storage Key"

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3564]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS2.1.2; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows 10 Enterprise (Version 1903, Build: 18362.356)
MAS Violated: MAS2.1.2

**Note:**
**Similar issue is observed even in "Link New HDInsight Cluster (Dark Theme)"**
**User is able to navigate to the next control using 'Ctrl+Tab' but user is not provided with this information**

**Repro Steps:**
1.Open Java EE - Eclipse application
2.Install 'Azure Toolkit for Eclipse' from Marketplace
3.Open Eclipse and install  the Scala plug-in 
4.Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
5.Tab till Windows menu option-->Show View-->Other-->Expand Azure-->Azure Explorer and press OK
6.Tab till HDInsight and press right click. Tab till 'Link a Cluster' option and press enter
7.Tab till "Storage Key" multiline text field
8.Verify whether keyboard trap is observed in the "Storage Key" multiline text field or not

**Actual Result:**
Keyboard trap is observed in the multi line text field "Storage Key"

**Expected Result:**
Keyboard focus should go to the next control "Storage Container" editable combo box after tabbing from multi line text field "Storage Key"

**User Impact:**
Keyboard users will find difficulty in navigating to the control "Storage Container" editable combo box, if there is a keyboard trap in multi line text field "Storage Key

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={ebdc3fe1-09a9-4aa2-a29b-e97764774e6c}

[Bug1_Keyboard trap in Storage key multi line text field.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635605/Bug1_Keyboard.trap.in.Storage.key.multi.line.text.field.pptx)

**Comment by t-rufang** [comment]
Note: When focus is at `Storage Key`, press tab, focus can't go to `Storage Container`, instead focus still stays at `Storage Key`.
![image](https://user-images.githubusercontent.com/32627233/65828125-0e23a900-e2ca-11e9-9e10-850d509d4a00.png)

**Comment by t-rufang** [comment]
`TextArea` component is kind of special, you need to press `ctrl tab` to escape the component and move to the next component and press `ctrl shift tab` to move to the prevous component.

**Comment by Tanushagottipati** [comment]
@Alexandria-R
Could you please suggest what action can be taken on bug. Here, user for user to navigate from text area control user has to press Control+tab

**Comment by Alexandria-R** [comment]
As long as this issue is well documented I do not believe it would be blocking. Please close the bug on that condition.

**Comment by t-rufang** [comment]
We have already add note in our toolkit document. Please find details here:  https://docs.microsoft.com/en-us/azure/hdinsight/spark/apache-spark-eclipse-tool-plugin#link-a-cluster

![image](https://user-images.githubusercontent.com/32627233/69393487-3d77f480-0d14-11ea-9a26-a5f87992bf14.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749732
